src/main/java/core/basesyntax/utils/CalculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.utils;
+
+public interface CalculateArea {
```

Comment:
better interface name. google about java interface naming advices
```suggestion
public interface AreaCalculator {
```